tutorial/numpy-functions.md

```diff
@@ -0,0 +1,385 @@
+# Supported Numpy function by Pyccel
+
+In Pyccel we try to support the Numpy functions which developers use the most.. Here are some of them:
+
+## [Norm](https://numpy.org/doc/stable/reference/generated/numpy.linalg.norm.html)
```

Comment:
Here `norm` should not be capitalized, because the name of a function should always be reported verbatim (Python is case-sensitive)